src/main/java/org/folio/search/service/setter/item/ItemPublicNotesProcessor.java

```diff
@@ -21,4 +22,12 @@
       .filter(CollectionUtils::isNotEmpty)
       .flatMap(Collection::stream);
   }
+
+  @Override
+  protected Stream<CirculationNote> getCirculationNotes(Instance instance) {
```

Comment:
Can you add one more case into a related unit test for the item with notes and circulationNotes?